src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int MAX_SIZE = 11;
+
+    public Figure getRandomFigure() {
+        Figure retVal = null;
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        Figures figure = Figures.values()[random.nextInt(Figures.values().length)];
+        switch (figure) {
+            case SQUARE: {
+                retVal = new Square(colorSupplier.getRandomColor(),
```

Comment:
```suggestion
                return new Square(colorSupplier.getRandomColor(),
```
можна з switch case одразу повертати